spec/feature/tutoring_session_scheduler_spec.rb

```diff
@@ -1,211 +1,219 @@
-      tsession.users << tutor
-
-      expect(TutoringSession.all.count).to eq(1)
-      visit("/tutoring_session/#{tsession.id}")
-      accept_confirm do
-        find(:link_or_button, 'delete').click
-      end
-      expect(TutoringSession.all.count).to eq(0)
-    end
-    
-    it 'should be able to delete session and any repeating sessions at the same time', :js => true do 
-      tsession = TutoringSession.create(:scheduled_datetime => scheduled_datetime, tutor_id: tutor.id)
-      tsession.users << tutor
-      tsession2 = TutoringSession.create(:scheduled_datetime => scheduled_datetime + 1.week, tutor_id: tutor.id)
-      tsession2.users << tutor
-      
-      expect(TutoringSession.all.count).to eq(2)
-      visit('/tutoring_session/' + tsession.id.to_s)
-      accept_confirm do
-        find(:link_or_button, 'delete self and following repeats').click
-      end
-      expect(TutoringSession.all.count).to eq(0)
-    end
-  end
-
-  describe 'UPDATE' do
-    it 'should be able to edit session details' do 
-      tsession = TutoringSession.create(:scheduled_datetime => scheduled_datetime, tutor_id: tutor.id)
-      tsession.users << tutor
-
-      visit("/tutoring_session/#{tsession.id}")
-      find(:link_or_button, 'edit').click
-
-      expect(page).to have_content('Edit Tutoring Session')
-      fill_in 'tutoring_session_scheduled_datetime', with: (scheduled_datetime + 1.hour)
-      find(:link_or_button, 'Update Tutoring session').click
-
-      expect(TutoringSession.all.count).to eq(1)
-      expect(TutoringSession.first.users.count).to eq(1)
-      expect(TutoringSession.first.scheduled_datetime).to eq((scheduled_datetime + 1.hour))
-    end
-
-    it 'should error on missing scheduled_datetime edit session details' do 
-      tsession = TutoringSession.create(:scheduled_datetime => scheduled_datetime, tutor_id: tutor.id)
-      tsession.users << tutor
-
-      visit("/tutoring_session/#{tsession.id}")
-      find(:link_or_button, 'edit').click
-
-      expect(page).to have_content('Edit Tutoring Session')
-      fill_in 'tutoring_session_scheduled_datetime', with: ''
-      find(:link_or_button, 'Update Tutoring session').click
-      expect(page).to have_content('Edit Tutoring Session')
-
-      expect(TutoringSession.all.count).to eq(1)
-    end
-  end
-end
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe TutoringSessionController, type: :feature do
+  let(:frozen_time) { '25 May 02:00:00 +0000'.to_datetime }
+  let(:tutor) { User.where(first_name: 'Tutor', last_name: 'User').first }
+  let(:scheduled_datetime) { '26 May 2021 08:00:00 +0000'.to_datetime }
+  let(:beginning_of_week) { Date.today.beginning_of_week }
+
+  after { Timecop.return }
+
+  before do
+    Timecop.freeze(frozen_time)
+    User.create(first_name: 'Admin', last_name: 'User', password: 'T3st!!a',
+                email: 'admin@tamu.edu')
```

Comment:
Couldn't lines 14/15 replace the part in the brackets on line 6? It'd be more intuitive, and it would result in fewer database interactions
```suggestion
  let!(:tutor) { User.create(first_name: 'Admin', last_name: 'User', password: 'T3st!!a',
                                       email: 'admin@tamu.edu') }
  let(:scheduled_datetime) { '26 May 2021 08:00:00 +0000'.to_datetime }
  let(:beginning_of_week) { Date.today.beginning_of_week }

  after { Timecop.return }

  before do
    Timecop.freeze(frozen_time)
```